src/Tribe/Views/V2/View.php

```diff
@@ -1849,19 +1864,27 @@ protected function get_breadcrumbs() {
 		$taxonomy    = TEC::TAXONOMY;
 		$context_tax = $context->get( $taxonomy, false );
 
-		// Get term slug if taxonomy is not empty
+		// Get term slug if taxonomy is not empty.
 		if ( ! empty( $context_tax ) ) {
-			$term  = get_term_by( 'slug', $context_tax, $taxonomy );
-			if ( ! empty( $term->name ) ) {
-				$label = $term->name;
+			$context_tax = (array) $context_tax;
+			foreach( $context_tax as $term_slug ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after FOREACH keyword; 0 found
* :no_entry_sign: There must be a space between foreach and the first parenthesis '('

<sub>hash: 8124d29e0533ca695fbf55c90873385e</sub>